### PR TITLE
chore: eslint format with prettier rules

### PR DESCRIPTION
**Pull request by QuiiBz**
Run Prettier rules when running ESLint, not sure if it was intended to not have it?

### CODE REVIEW

**Review by Hacksore — APPROVED** [review]

**Comment by QuiiBz** [comment]
Yeah let's wait for Anthony's answer

**Review by bautistaaa — APPROVED** [review]

**Comment by Hacksore** [comment]
Don't merge until @anthonyshew weighs in.

**Comment by jhechtf** [comment]
do we really have to use prettier though :c

**Comment by Hacksore** [comment]
> do we really have to use prettier though :c

img

**Comment by jhechtf** [comment]
> 

that does not answer any questions.

**Comment by QuiiBz** [comment]
> that does not answer any questions.

Why wouldn't we want to use a formatter, without even having to think about it? It would avoid format errors in the CI and make sure we have good consistency in our codebase.

**Comment by jhechtf** [comment]
Because I'm not a fan of most of the things that Prettier enforces and have found that consistent ESLint rules tend to remove the need for Prettier. 

And then people integrate Prettier into ESLint, which just seems like wearing socks on top of your socks to me.

**Comment by Hacksore** [comment]
Let's close this for now and reopen if @anthonyshew can tell us if we should do this.

**Comment by anthonyshew** [comment]
TL;DR: We don't need this PR. :)

> Pasting here from Discord!

eslint-plugin-prettier: Runs Prettier using ESLint which is slower than Prettier running Prettier. Additionally, we’d be running the Prettier formatting twice so doing unnecessary work.

eslint-config-prettier: Prevent running rules in ESLint that can overlap with your Prettier configuration. This is good, we hate when the formatter and the linter disagree and/or could potentially disagree.

Good news is: We don’t have to handle it ourselves because @vercel/style-guide already has eslint-config-prettier in it here: https://github.com/vercel/style-guide/blob/8ef6fc54e91092ab4288dee9588c2e70d8087322/package.json#L37
